### DISCUSSION TITLE
Why is a signal that is reflected once weaker than a signal that is reflected three times?

### DISCUSSION THREAD

**Discussion by hooonji** [NVlabs/sionna#594] [Q&A]
![스크린샷 2024-09-24 154405](https://github.com/user-attachments/assets/d57e97c4-c193-4d67-a3d6-8dcf5d86734f)

I'm doing a test to see the difference in signal strength based on the number of reflections. 

I'm running into a situation that I don't understand, so I thought I'd ask.

I followed the path of increasing the number of reflections in the same situation, with the same settings, and compared the coefficient values.

But the results are strange, 2 reflections have a larger coeff than 1 reflection.

The total length of the path is also longer for the path with 2 reflections, so I don't know how this can happen.

My expectation is that the coeff should be smaller with more reflections unless there is a really unusual situation, but I don't understand why this happens.

In addition to the above situation, when i actually checked the coeff of the path reflected 4 times, i found that it was larger than the value reflected 1 time.

If I'm misunderstanding something, I'd really appreciate it if you could let me know.

Thank you so much for reading.

**Comment by jhoydis** [comment]
Hi @hooonji,

That's an interesting test. We would need the full code to reproduce your results in order to see what is going on, ideally using one of the built-in scenes.

Do you use isotropic antennas? Which polarization do you use for the tx and rx antennas? 

It could in principle happen that a reflected path is stronger than the LoS path if transmitter and receiver have directive antennas and dot not "look" at each other. The same holds for paths consisting of multiple reflections.
